test/src/org/labkey/test/tests/snd/SNDTest.java

```diff
@@ -1740,9 +1740,6 @@ public void testProjectApis() throws Exception
         runScriptExpectedFail(reviseProjectApi(TEST_PROJECT_ID, 2, "2018-02-15", null, "description", TEST_REV_PROJECT_DESC));
         runScript(editProjectApi(TEST_PROJECT_ID + 1, 0, "projectItems", null));
 
-        // Expected QUS delete row errors
-        checkExpectedErrors(6);
```

Comment:
Why are these being deleted? Do the changes above result in no errors being generated somehow?

Reply:
According to Matt B we are no longer logging QUSExceptions.

Reply:
This is not related to the other changes here, just cleaning up TC failures.